L1Trigger/TrackFindingTracklet/interface/CandidateMatchMemory.h

```diff
@@ -0,0 +1,129 @@
+//Holds the candidate matches
+#ifndef CANDIDATEMATCHMEMORY_H
+#define CANDIDATEMATCHMEMORY_H
+
+#include "Tracklet.h"
+#include "MemoryBase.h"
+#include "Stub.h"
+#include "L1TStub.h"
+
+using namespace std;
```

Comment:
Please don't `using namespace std` in a header file.

Reply:
fixed by migrating to h/cc files